src/software/simulation/er_force_simulator.h

```diff
@@ -124,6 +126,16 @@ class ErForceSimulator
      */
     void resetCurrentTime();
 
+    /**
+     * Creates the default realism config
+     */
+    static std::unique_ptr<RealismConfigErForce> createDefaultRealismConfig();
```

Comment:
nit: to stay consisten with (most) the other methods here, `@return` should be added to the Javadoc. Similar to the Javadoc for `getSSLWrapperPackets` it can explain what the returned proto is used for (i.e. passed to the er force simulator to configure the realism...)
Same with `createRealisticRealismConfig`